.github/workflows/integration-test.yml

```diff
@@ -35,13 +63,33 @@ jobs:
           echo "plugins.security.unsupported.restapi.allow_securityconfig_modification: true" >> ./opensearch-${{ env.OPENSEARCH_VERSION }}-SNAPSHOT/config/opensearch.yml
           EOF
 
+      #- name: Create Setup Script for Windows
+      #  if: ${{ runner.os == 'Windows' }}
+      #  run: |
+      #    New-Item .\setup.bat -type file
+      #    Set-Content .\setup.bat -Value "powershell.exe .\opensearch-${{ env.OPENSEARCH_VERSION }}-SNAPSHOT\plugins\${{ env.PLUGIN_NAME }}\tools\install_demo_configuration.bat -y"
+      #    Add-Content -Path .\setup.bat -Value "echo plugins.security.unsupported.restapi.allow_securityconfig_modification: true >> .\opensearch-${{ env.OPENSEARCH_VERSION }}-SNAPSHOT\config\opensearch.yml"
+      #    Get-Content .\setup.bat
+      
+      - name: Create Setup Script for Windows
+        if: ${{ runner.os == 'Windows' }}
+        run: |
+          New-Item .\setup.bat -type file
+          Set-Content .\setup.bat -Value "powershell.exe -noexit -command `"echo 'y' | .\opensearch-${{ env.OPENSEARCH_VERSION }}-SNAPSHOT\plugins\${{ env.PLUGIN_NAME }}\tools\install_demo_configuration.bat`""
```

Comment:
Awesome to see that the cypress tests are now running 🙌 .

This line can be switched to using the options of the batch script. Instead of `echo 'y' |` to answer the prompts, the prompts can automatically be answered by adding the options `-y -i -c` after `install_demo_configuration.bat`

Reply:
Got it! I will update that~

Reply:
I just fixed, and it also fixed the error : https://github.com/opensearch-project/security-dashboards-plugin/actions/runs/3999286312/jobs/6862995899